apps/website/src/pages/docs/buttongroup.mdx

```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='buttons-buttongroup' />
-
 <LiveExample src='ButtonGroup.main.tsx'>
   <AllExamples.ButtonGroupMainExample client:load />
 </LiveExample>
 
-A button group consists of similar tools. The button group can only consist of default buttons or borderless buttons, a button group can not be a mix of the two. If there is not enough space to display all of the buttons, the buttons can truncate into an overflow menu.
+Use a button group if you want to but similar functionality in one place. It helps users differentiate functional areas of the UI.
+
+Buttons in button group can be active, disabled and both, active and disabled.
+
+## Appearance
+
+### Overflow
+
+If there is not enough space to display all of the buttons, the buttons can truncate into an overflow menu.
+
+<LiveExample src='ButtonGroup.overflow.tsx'>
+  <AllExamples.ButtonGroupOverflowExample client:load />
+</LiveExample>
+
+### Vertical
+
+Use <code>orientation</code> prop to switch button group to vertical.
+
+<LiveExample src='ButtonGroup.vertical.tsx'>
+  <AllExamples.ButtonGroupVerticalExample client:load />
+</LiveExample>
+
+### With input
+
+You can combine [inputs](input) with buttons in a button group.
+
+<LiveExample src='ButtonGroup.input.tsx'>
+  <AllExamples.ButtonGroupInputExample client:load />
+</LiveExample>
+
+## Usage
+
+- The button group should only consist of default buttons or borderless buttons
+- You should not mix different styles buttons in the same group.
```

Comment:
```suggestion
- You should not mix buttons with different styles in the same group.
```

OR

`You should not mix different styled buttons in the same group.`

Reply:
Updated to OR version